clients/client-go/tcgithub/types.go

```diff
@@ -110,6 +112,98 @@ type (
 	// Syntax:     ^[a-zA-Z0-9]{8}-[a-zA-Z0-9]{4}-[a-zA-Z0-9]{4}-[a-zA-Z0-9]{4}-[a-zA-Z0-9]{12}$
 	GithubGUID string
 
+	// Render .taskcluster.yml for given event.
+	RenderTaskclusterYmlInput struct {
```

Comment:
Let's consistently use yaml or yml if possible. Further up we have RenderTaskclusterYaml. If we go with yml instead of yaml (I don't mind which) then let's add that as a common initialism instead. But maybe I prefer YAML to YML.